extensions/amp-story/1.0/amp-story-draggable-drawer-header.css

```diff
@@ -74,3 +74,75 @@
 .i-amphtml-story-draggable-drawer-header-attachment-remote {
   display: none !important;
 }
+
+/* For amp-story-page-attachment-ui-v2 experiment. */
+
+:host {
+  border-radius: inherit !important; /* Propogate border-radius */
+}
+
+.amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header {
+  justify-content: center !important;
+  box-shadow: none !important;
+  border-radius: inherit !important;
+  height: auto !important;
+  position: sticky !important;
+  top: 0 !important;
+}
+
+:not([desktop]).amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header {
+  height: 20px !important;
+}
+
+.amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header .i-amphtml-story-draggable-drawer-header-title-and-close {
+  opacity: 1 !important;
+  height: 44px !important;
+  /* line-height: 44px!important; */
+  width: 100% !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  background: white !important;
+  border-radius: inherit !important;
+}
+
+:not([desktop]).amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header .i-amphtml-story-draggable-drawer-header-title-and-close {
+  position: absolute !important;
+  transition: opacity .3s !important;
+  opacity: 0 !important;
+}
+
+:not([desktop]).amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header .i-amphtml-story-draggable-drawer-header-title-and-close:focus-within {
+  opacity: 1 !important;
+}
+
+:not([desktop]).amp-story-page-attachment-ui-v2.i-amphtml-story-draggable-drawer-header.i-amphtml-story-draggable-drawer-header-stuck .i-amphtml-story-draggable-drawer-header-title-and-close {
+  opacity: 1 !important;
+}
+
+.amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-title {
+  font-family: 'Poppins', sans-serif !important;
```

Comment:
Wouldn't we need to import Poppins programmatically for this to work?

Reply:
Good catch. I'll remove this until we discuss and have a plan with the team.